mangaki/mangaki/tests/test_reco.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+
+class RecoTest(TestCase):
+
+    def setUp(self):
+        User.objects.create_user(username='test', password='test')
+
+    def test_reco(self, **kwargs):
+        self.client.login(username='test', password='test')
+        reco_url = reverse('get-reco-algo-list', args=['svd', 'all'])
+        self.assertEqual(reco_url, '/data/reco/svd/all.json')
```

Comment:
I'd say that this test is not useful for Mangaki's code.

You are currently testing if the Django's router is able to return the current URL of our `urls.py` file.

But this is Django's code, not ours. Django's code is already tested.